Dockerfile

```diff
@@ -7,29 +7,47 @@ FROM $BASE_OS:$BASE_CODENAME AS build-stage
 RUN <<EOR
 	set -eu
 
+	# Workaround for outstanding fix of https://bugs.launchpad.net/ubuntu/+source/python-build/+bug/1992108
+	if grep -q ^UBUNTU_CODENAME=jammy /etc/os-release; then
+		echo >>/etc/apt/sources.list.d/jammy-proposed.list 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/     jammy-proposed universe'
```

Comment:
Presuming this fixed the issues, poking https://bugs.launchpad.net/ubuntu/+source/python-build/+bug/1992108 and telling them it works fine, perhaps with link to this PR, should hopefully resolve the block.

Reply:
Done